### DISCUSSION TITLE
How can I create a batch of images from one prompt, making sure each image looks different and not too similar?

### DISCUSSION THREAD

**Discussion by TechMitten** [lllyasviel/stable-diffusion-webui-forge#1907] [Q&A] [answered]
I’m trying to create batches of 25, 50, or more images at once, where each image is unique but follows the same style from the prompt. For example, if I generate images based on words like 'serene,' 'peaceful,' and 'calm,' they all end up being pictures of rocks in water. I’d like to see more variety in the results while keeping the same overall theme.

**Comment by WalternateBishop** [comment, marked as answer]
you can try adding wildcards, I've found that this extension works great with forge: https://github.com/adieyal/sd-dynamic-prompts

**Reply by TechMitten** [reply]
> you can try adding wildcards, I've found that this extension works great with forge: https://github.com/adieyal/sd-dynamic-prompts

Thank you
